imageio/imageio-psd/src/main/java/com/twelvemonkeys/imageio/plugins/psd/PSDLayerInfo.java

```diff
@@ -161,6 +178,30 @@
         if (pInput.getStreamPosition() != expectedEnd) {
             pInput.seek(expectedEnd);
         }
+
+        if(sectionDividerSettingType == SectionDividerSetting.BOUNDING_SECTION_DIVIDER){
+            if(group == null){
+                group = new LinkedList<>();
+                groupStack.add(group);
+            } else {
+                groupStack.add(group);
+                group = new LinkedList<>();
+            }
+            isSectionDivider = true;
+        } else if(sectionDividerSettingType == SectionDividerSetting.OPEN_FOLDER ||
+                sectionDividerSettingType == SectionDividerSetting.CLOSED_FOLDER){
+            for(PSDLayerInfo info : group){
+                info.groupLayerId = this.layerId;
+            }
+            group = groupStack.pop();
+            group.add(this);
+            this.isGroup = true;
+        } else {
+            if(group != null){
+                group.add(this);
+            }
+        }
```

Comment:
Please reformat this code block using the correct code style.

Also, the usage of `static` variables will not work if reading multiple PSD images at the same time.